File: Python/card.py
from Payment import Payment 

class Card(Payment):
    id = int
    card_name = str
    card_number = int
    ccv = int
    expiration_date = int
    address = str
    
    def __init__(self, id, card_name, card_number, ccv, expiration_date, address):
        
        super().__init__(id)
        self.card_name = card_name
        self.card_number = card_number
        self.ccv = ccv
        self.expiration_date = expiration_date
        self.address = address

File: Java/Card.java
public class Card extends Payment{

    Integer numberCard;
    String dateCard;
    String cvv;
    String address;

    public Card(Integer id, Integer numberCard, String dateCard, String cvv, String address){
        super(id);
        this.numberCard = numberCard;
        this.dateCard = dateCard;
        this.cvv = cvv;
        this.address = address;
    }
    
}
